CMakePresets.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": 4,
+    "version": 3,
```

Comment:
question: Is this change related to this fix? Is this a local change you'd like to keep just for you and should be removed from this PR?